docker/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+suricata-update && suricata $@
```

Comment:
I did not find any edit of the suricata.yaml in suricata 4.0.x this means the classical structure of the ruleset is used (import of list of files present in the ET ruleset). So the `suricata.rules` file created by `suricata-update` is unused.

Things should change in 4.1.x as suricata.rules should be activated if you have suricata-update.

Reply:
Amended to add `-S /var/lib/suricata/rules/suricata.rules` in the entrypoint.